### DISCUSSION TITLE
"conda env create -f environment-ufsreplay.yml" stuck in "Solving environment:"

### DISCUSSION THREAD

**Discussion by AnningCheng-NOAA** [NCAR/ccpp-scm#390] [Cases, Initial Conditions, Forcing] [answered]
Has anybody have the issue of running "conda env create -f environment-ufsreplay.yml", stuck in "Solving environment:" for hours?  IT happens as I checkout the SCM  as "git clone --recursive https://github.com/NCAR/ccpp-scm"

**Comment by dustinswales** [comment, marked as answer]
@AnningCheng-NOAA  
 I just tried on Cheyenne w/o issue:
![Screenshot 2023-06-21 at 9 40 18 AM](https://github.com/NCAR/ccpp-scm/assets/13949033/81dd2204-29e1-42de-9625-1ac32f5f6adc)
But on Hera got hung up "Solving environment"...
I had no problem building the environment manually on Hera,
> conda create -n env_ufsreplay python=3.8.5
> conda activate env_ufsreplay
> conda install -c conda-forge netcdf4
> conda install -y -c conda-forge f90nml
> conda install -y -c conda-forge numpy
> conda install -y -c conda-forge xarray
> conda install -y -c conda-forge shapely
> conda install -y -c conda-forge xesmf
@mkavulich @hertneky

**Reply by hertneky** [reply]
@dustin I had also had this exact issue on Hera, but had no issue building it manually as you have shown here.

**Comment by AnningCheng-NOAA** [comment]
I do not have an account in  Cheyenne. It seems like that the package has
not been tested in HERA yet.
